Pascal/Pascal.sublime-syntax

```diff
@@ -47,7 +47,7 @@ contexts:
             0: punctuation.definition.comment.pascal
           push:
```

Comment:
changing this `push` to a `set` will fix the problem, no need to stop capturing `\n` as part of a comment